eth/monitor/monitor.go

```diff
@@ -0,0 +1,377 @@
+// Copyright 2018-2020 Celer Network
+
+package monitor
+
+import (
+	"container/heap"
+	"encoding/hex"
+	"fmt"
+	"math/big"
+	"math/rand"
+	"sync"
+	"time"
+
+	"github.com/celer-network/goutils/eth"
+	"github.com/celer-network/goutils/eth/watcher"
+	"github.com/celer-network/goutils/log"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+const (
+	// default log watch polling as a multiplier of block number if not specified
+	// ie. 1 means check log every block
```

Comment:
here to be more accurate we should say check log every blockIntervalSec
and multiplier of blockIntervalSec

Reply:
updated